core/settings/qPrefCloudStorage.cpp

```diff
@@ -28,6 +30,10 @@ void qPrefCloudStorage::loadSync(bool doSync)
 	disk_save_password_local(doSync);
 	disk_save_userid_local(doSync);
 	disk_userid(doSync);
+	qDebug() << "baseurl: " << QString(prefs.cloud_base_url) << " ok\n";
+	qDebug() << "giturl: " << QString(prefs.cloud_git_url) << " ok\n";
+	qDebug() << "git local: " << (prefs.git_local_only ? "true" : "false") << " ok\n";
+
```

Comment:
If any of these paths should be reported with '-v', then let's do that there (we do report the 'cloud URL' already)